concert/app/src/functions/on_choice_made/index.py

```diff
@@ -6,10 +6,10 @@
 Processes websocket API messages from API Gateway
 """
 
-AGGREGATE_CHOICE_DELIVERY_STREAM_NAME = os.environ.get(
+AGGREGATE_CHOICE_DELIVERY_STREAM_NAME = os.environ[
     "AGGREGATE_CHOICE_DELIVERY_STREAM_NAME"
-)
-CHOICE_MADE_SNS_TOPIC_ARN = os.environ.get("CHOICE_MADE_SNS_TOPIC_ARN")
+]
+CHOICE_MADE_SNS_TOPIC_ARN = os.environ["CHOICE_MADE_SNS_TOPIC_ARN"]
```

Comment:
Why this change? The next one still uses `.get()`

Reply:
Yeah, this should still be `.get()`; don't think this was intentional. Fixed. Thanks for catching.